lib/3llo/presenter/help.rb

```diff
@@ -28,6 +28,7 @@ def menu_text
     card show <card_id>         - Show card information
     card move <card_id>         - Move card to a list
     card self-assign <card_id>  - Self-assign a card
+    card assign <card_id>       - Assign a card to a user
```

Comment:
I am not a native English speaker but was wondering whether it would be more natural to say "Assign a card to a user" or "Assign a user to a card"?

Reply:
Me too but yours look right.